plugins/generator-1.19.2/forge-1.19.2/templates/livingentity/livingentity_renderer.java.ftl

```diff
@@ -107,6 +107,34 @@ public class ${name}Renderer extends <#if humanoid>Humanoid</#if>MobRenderer<${n
 			}
 		});
 		</#if>
+
+		<#if data.modelLayers??>
+			<#list data.modelLayers as layer>
+				this.addLayer(new RenderLayer<${name}Entity, ${model}>(this) {
+					@Override
+					public void render(PoseStack poseStack, MultiBufferSource bufferSource, int light, ${name}Entity entity, float limbSwing, float limbSwingAmount, float partialTicks, float ageInTicks, float netHeadYaw, float headPitch) {
+						<#if hasProcedure(layer.condition)>
+							Level world = entity.level;
+							double x = entity.getX();
+							double y = entity.getY();
+							double z = entity.getZ();
+						</#if>
+						if (<@procedureOBJToConditionCode layer.condition/>) {

```

Comment:
You don't have to generate code for this condition if it is left empty.

Reply:
But that means I also have to check if there is a condition at the bottom where the if closes... Overlays leave the if true anyway if there isn't a condition so it should be fine right?

Reply:
changed it anyway